conans/requirements.txt

```diff
@@ -7,8 +7,8 @@ fasteners>=0.14.1
 six>=1.10.0
 node-semver==0.2.0
 distro>=1.0.2, <1.2.0
-pylint>=1.8.1, <1.9.0
+pylint
 future==0.16.0
 pygments>=2.0, <3.0
-astroid>=1.6, <1.7
+astroid
```

Comment:
Just thought: Conan doesn't depend on astroid directly (except for plugin). Does it still need to be specified in requirements or could we just let pip figure out what's needed?

Reply:
As the plugin is importing "astroid" it should declare it as requirement.